atlassian_jwt_auth/contrib/flask_app/utils.py

```diff
@@ -0,0 +1,30 @@
+from flask import current_app
+import jwt.exceptions
+import requests.exceptions
+
+
+def parse_jwt(verifier, encoded_jwt):
+    """Decode an encoded JWT using stored config."""
+    try:
+        claims = verifier.verify_jwt(
+            encoded_jwt,
+            current_app.config.asap['VALID_AUDIENCE'],
+            leeway=60,
+            verify=current_app.config.asap.get('VERIFY_TLS_CERT', True)
```

Comment:
Obviously the token should be a required parameter. Valid audience seems like something we definitely want (I can't think of a reason why an ASAP service wouldn't want to validate that a token wasn't being re-used across services). It might make sense to customize leeway.

Yet again, `VERIFY_TLS_CERT` was part of the original copy & paste from the Instant Micros template. I think the main reasoning in its existence was for testing situtations. Its a significant amount of extra work to spin up a full TLS server to simulate a key repository but with my testing utils class that I added, we probably don't need this at all anymore. I'd be open to removing it